<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FooterPrst</class>
 <widget class="QWidget" name="FooterPrst">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>35</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>19</height>
      </size>
     </property>
     <property name="text">
      <string>Bank</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="SpinBox" name="spinBank">
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>23</height>
      </size>
     </property>
     <property name="accelerated">
      <bool>true</bool>
     </property>
     <property name="maximum">
      <number>128</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>19</height>
      </size>
     </property>
     <property name="text">
      <string>Preset</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="SpinBox" name="spinPreset">
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>23</height>
      </size>
     </property>
     <property name="accelerated">
      <bool>true</bool>
     </property>
     <property name="maximum">
      <number>127</number>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>1</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="labelPercussion">
     <property name="text">
      <string>(percussion)</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SpinBox</class>
   <extends>QSpinBox</extends>
   <header>footerprst.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
